pointcloud_boxcrop/CMakeLists.txt

```
@@ -0,0 +1,55 @@
+cmake_minimum_required(VERSION 3.8)
+project(pointcloud_boxcrop)
```

On jazzy I have deprecated info about using older than 3.10 cmake version
```suggestion
cmake_minimum_required(VERSION 3.11)
project(pointcloud_boxcrop)
```